src/status_im/ui/components/common/styles.cljs

```diff
@@ -137,10 +134,9 @@
    :align-items      :center
    :justify-content  :center})
 
-(defnstyle counter-label [size]
+(defn counter-label [size]
   {:font-size  (/ size 2)
    :color      colors/white
-   :android    {:line-height (+ (/ size 2) 2)}
```

Comment:
won't removing that break things? it looked like it did something important xD

Reply:
the line height is automatically added to text fields based on the font-size now